src/Portal/resources/coremod.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
What are those *.json files in resources for?

Reply:
Good catch, we only need one of them for tests